installer/x86_64/install.sh

```diff
@@ -485,7 +514,14 @@ fi
 
 # Add a menu entry for the DEMO OS
 # Note: assume that apparmor is supported in the kernel
-demo_grub_entry="$demo_volume_revision_label"
+demo_grub_entry="$demo_volume_revision_label-${git_revision}"
+if [ "$environment" = "sonic" ]; then
+    running_sonic_revision=$(cat /etc/sonic/sonic_version.yml | grep build_version | cut -f2 -d" ")
```

Comment:
>running_sonic_revision [](start = 4, length = 22)

Instead of finding the running revision, we can easily find the default entry in grub.cfg. So it is runnable both in ONIE and in sonic, you don't need if-condition.